sql/driver/doc.go

```diff
@@ -235,6 +238,45 @@ Querying rows:
 
 	SELECT __key, name FROM person WHERE age > 30
 
+2) "json" is a first-class SQL type with IMap and Kafka Connector support.
+
+Creating a mapping:
+
+	CREATE MAPPING person (
+		__key BIGINT,
+		this  JSON
+	)
+	TYPE IMAP
+	OPTIONS (
+		'keyFormat' = 'bigint',
+		'valueFormat' = 'json'
+	)
+
+Inserting rows:
+
+	// Use serialization.JSON type to specify a JSON value.
+	INSERT INTO person VALUES(100, serialization.JSON(fmt.Sprintf(`{"age":%d, "name":%s}`, 35, 'Jane Doe')))
+
+Querying rows:
+
+	// Use serialization.JSON type to scan JSON string.
+	q := fmt.Sprintf(`SELECT this FROM "%s" WHERE CAST(JSON_VALUE(this, '$.age') AS DOUBLE) > ?`, mapName)
+	rows, err := db.Query(q, minAge)
+	for rows.Next() {
+		var js serialization.JSON
+		rows.Scan(&js)
+	}
+
+Supported JSON related operations:
+
+1. `JSON_QUERY(jsonArg VARCHAR|JSON, jsonPath VARCHAR ... <extended syntax> ): returns JSON`: returns a json-object/array by
```

Comment:
Backtick character has no meaning for godoc.

Reply:
Also, I think there is no need for "returns JSON" text in the documentation above. There is already an explanation about that. How about something like this:
```
JSON_QUERY returns a json-object/array by the given JSON path.

    JSON_QUERY(jsonArg VARCHAR|JSON, jsonPath VARCHAR ... <extended syntax>)
```